spec/feature_tests/user_stories_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'menu'
+require 'order'
+require 'send_sms'
+
+# As a customer
+# So that I can check if I want to order something
+# I would like to see a list of dishes with prices
+describe Menu do
```

Comment:
These tests look like a mix between feature tests (testing functionality from the users perspective) and unit tests (testing that a particular unit of code e.g. a method on a class, does what is expected.
I recommend splitting these out, so you have a clear division between feature tests and unit tests.